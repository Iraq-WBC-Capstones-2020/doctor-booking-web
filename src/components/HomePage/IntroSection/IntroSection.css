@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
* {
  font-family: 'Roboto', sans-serif;
}

.introSection {
  position: relative;
  margin-bottom: 100px;
}

.penIntro {
  position: absolute;
  z-index: -1;
  top: -20%;
  left: 00%;
}

h1 {
  color: #026fc2;
  font-size: 36px;
  font-weight: 500;
}

p {
  color: #9dafbd;
}

.IntroSection__image {
  width: 100%;
}
/*# sourceMappingURL=IntroSection.css.map */
