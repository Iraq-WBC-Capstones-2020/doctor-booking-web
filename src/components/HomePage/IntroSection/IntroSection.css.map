{
    "version": 3,
    "mappings": "AAAA,OAAO,CAAC,gFAAI;AACZ,AAAA,CAAC,CAAC;EACA,WAAW,EAAE,oBAAoB;CAClC;;AAED,AAAA,aAAa,CAAC;EACZ,QAAQ,EAAE,QAAQ;EAClB,aAAa,EAAE,KAAK;CACrB;;AAED,AAAA,SAAS,CAAC;EACR,QAAQ,EAAE,QAAQ;EAClB,OAAO,EAAE,EAAE;EACX,GAAG,EAAE,IAAI;EACT,IAAI,EAAE,GAAG;CACV;;AAED,AAAA,EAAE,CAAC;EACD,KAAK,EAAE,OAAO;EACd,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,GAAG;CACjB;;AAED,AAAA,CAAC,CAAC;EACA,KAAK,EAAE,OAAO;CACf;;AAED,AAAA,oBAAoB,CAAC;EACnB,KAAK,EAAE,IAAI;CACZ",
    "sources": [
        "IntroSection.scss"
    ],
    "names": [],
    "file": "IntroSection.css"
}