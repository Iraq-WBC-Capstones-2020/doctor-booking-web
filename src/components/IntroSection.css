@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
* {
  font-family: 'Roboto', sans-serif;
}

h1 {
  color: #026fc2;
  font-size: 36px;
  font-weight: 500;
}

p {
  color: #9dafbd;
}

.IntroSection__image {
  width: 100%;
}
