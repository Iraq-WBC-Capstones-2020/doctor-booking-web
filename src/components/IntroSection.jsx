import React from 'react';

function IntroSection() {
  return <div></div>;
}

export default IntroSection;
