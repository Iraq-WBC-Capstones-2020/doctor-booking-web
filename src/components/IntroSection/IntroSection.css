@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
* {
  font-family: 'Roboto', sans-serif;
}

.introSection {
  position: relative;
  margin-bottom: 150px;
  z-index: 1;
}

.penIntro {
  position: absolute;
  z-index: -1;
  top: -44%;
  left: 00%;
}

h1 {
  color: #026fc2;
  font-size: 36px;
  font-weight: 500;
}

p {
  font-size: 16px;
  letter-spacing: 1px;
  line-height: 28px;
  color: #7d7987;
  padding-right: 70px;
}

.IntroSection__image {
  width: 100%;
}

button {
  font-size: 14px;
  font-weight: 500;
  color: red;
}

#getInTouch {
  background-color: #f5f9fd;
}
/*# sourceMappingURL=IntroSection.css.map */
