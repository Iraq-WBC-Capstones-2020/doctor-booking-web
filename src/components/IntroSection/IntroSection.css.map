{
    "version": 3,
    "mappings": "AAAA,OAAO,CAAC,gFAAI;AACZ,AAAA,CAAC,CAAC;EACA,WAAW,EAAE,oBAAoB;CAClC;;AAED,AAAA,aAAa,CAAC;EACZ,QAAQ,EAAE,QAAQ;EAClB,aAAa,EAAE,KAAK;EACpB,OAAO,EAAE,CAAC;CACX;;AAED,AAAA,SAAS,CAAC;EACR,QAAQ,EAAE,QAAQ;EAClB,OAAO,EAAE,EAAE;EACX,GAAG,EAAE,IAAI;EACT,IAAI,EAAE,GAAG;CACV;;AACD,AAAA,EAAE,CAAC;EACD,KAAK,EAAE,OAAO;EACd,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,GAAG;CACjB;;AAED,AAAA,CAAC,CAAC;EACA,SAAS,EAAE,IAAI;EACf,cAAc,EAAE,GAAG;EACnB,WAAW,EAAE,IAAI;EACjB,KAAK,EAAE,OAAO;EACd,aAAa,EAAE,IAAI;CACpB;;AAED,AAAA,oBAAoB,CAAC;EACnB,KAAK,EAAE,IAAI;CACZ;;AAED,AAAA,MAAM,CAAC;EACL,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,GAAG;EAChB,KAAK,EAAE,GAAG;CACX;;AAED,AAAA,WAAW,CAAC;EACV,gBAAgB,EAAE,OAAO;CAC1B",
    "sources": [
        "IntroSection.scss"
    ],
    "names": [],
    "file": "IntroSection.css"
}