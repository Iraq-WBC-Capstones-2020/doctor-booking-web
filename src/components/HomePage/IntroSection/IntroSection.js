import React from 'react';
import { Row, Col, Button, Container } from 'react-bootstrap';
import penIntro from './img/penIntro.svg';
import intro from './img/intro.svg';
import './IntroSection.scss';
import '../HomePageGlobalStyle.scss';
import { useTranslation  } from 'react-i18next';
import i18next from 'i18next';

function IntroSection() {

  const { t, i18n } = useTranslation(['validation','common'])
 
  return (
    <div className="introSection">
      <img src={penIntro} className="penIntro" alt="" />
      <Container>
        <Row className=" my-5">
          <Col lg="6" className=" d-flex align-items-center">
            <div>
              <h1 className="my-3 sectionHeading">
              {t('validation:homePage.healthImportant.headLine.title')}{' '}
                <span className="d-block">{t('validation:homePage.healthImportant.headLine.titleSpan')}</span>
              </h1>
              <p className="sectionCaption">
              {t('validation:homePage.healthImportant.paragraph')}
              </p>
              <div className="my-4">
                <a href="#getInTouch">
                  <Button className="btn-lg sectionBtn" variant="primary">
                    Get Started
                  </Button>
                </a>
                <a href="#videoSection">
                  <Button
                    className="btn-lg ml-4 sectionBtn"
                    variant="outline-primary"
                  >
                   {t('common:introVideo')}
                  </Button>
                </a>
              </div>
            </div>
          </Col>
          <Col lg="6" className="">
            <img src={intro} className="img-fluid introSectionImage" alt="" />
          </Col>
        </Row>
      </Container>
    </div>
  );
}

export default IntroSection;
