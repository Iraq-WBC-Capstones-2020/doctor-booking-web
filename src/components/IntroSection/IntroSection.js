import React from 'react';
import { Row, Col, Button, Container } from 'react-bootstrap';
import penIntro from './img/penIntro.svg';
import intro from './img/intro.svg';
import './IntroSection.scss';

function IntroSection() {
  return (
    <div className="introSection">
      <img src={penIntro} className="penIntro d-none d-lg-block" alt="" />
      <Container>
        <Row className=" my-5">
          <Col lg="6" className=" d-flex align-items-center">
            <div>
              <h1 className="my-3">
                Your health is important{' '}
                <span className="d-block">for us.</span>
              </h1>
              <p className="textShow">
                This website help you to find the best and suit doctor for your
                situation , and save you alot of time for searching doctor
              </p>
              <div className="my-4">
                <a href="#getInTouch">
                  <Button className="btn" variant="primary">
                    Get Started
                  </Button>
                </a>
                <a href="#video">
                  <Button className="btn ml-4" variant="outline-primary">
                    Intro Video
                  </Button>
                </a>
              </div>
            </div>
          </Col>
          <Col lg="6" className="">
            <img src={intro} className="img-fluid IntroSection__image" alt="" />
          </Col>
        </Row>
      </Container>
    </div>
  );
}

export default IntroSection;
