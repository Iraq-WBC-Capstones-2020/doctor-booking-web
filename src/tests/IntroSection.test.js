import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import IntroSection from '../components/HomePage/IntroSection/IntroSection';

test('Intro section  is rendering  ', () => {
  render(<IntroSection />);
});

test('Intro  section run well ', () => {
  const tree = renderer.create(<IntroSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
