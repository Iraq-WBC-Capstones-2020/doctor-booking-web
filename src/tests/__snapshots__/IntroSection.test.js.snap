// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro  section run well  1`] = `
<div
  className="introSection"
>
  <img
    alt=""
    className="penIntro d-none d-lg-block"
    src="penIntro.svg"
  />
  <div
    className="container"
  >
    <div
      className=" my-5 row"
    >
      <div
        className=" d-flex align-items-center col-lg-6"
      >
        <div>
          <h1
            className="my-3"
          >
            Your health is important
             
            <span
              className="d-block"
            >
              for us.
            </span>
          </h1>
          <p
            className="textShow"
          >
            This website help you to find the best and suit doctor for your situation , and save you alot of time for searching doctor
          </p>
          <div
            className="my-4"
          >
            <a
              href="#getInTouch"
            >
              <button
                className="btn btn btn-primary"
                disabled={false}
                type="button"
              >
                Get Started
              </button>
            </a>
            <a
              href="#video"
            >
              <button
                className="btn ml-4 btn btn-outline-primary"
                disabled={false}
                type="button"
              >
                Intro Video
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        className="col-lg-6"
      >
        <img
          alt=""
          className="img-fluid IntroSection__image"
          src="intro.svg"
        />
      </div>
    </div>
  </div>
</div>
`;
